howso/engine/session.py

```diff
@@ -333,14 +315,14 @@ def get_session(
     """
     client = client or get_client()
     session = client.get_session(str(session_id))
-    return Session.from_openapi(session, client=client)
+    return Session.from_dict(session)
```

Comment:
`Session.from_dict` should be updated to take the `client` as an optional parameter. And all calls to it in this module should pass it along.

Reply:
Should be addressed by: d66654ea10038f70a275bb4aa504128552745e60
and 2c0b79b34673cc4511cda3aa80964b7e41a0dcdd